src/xenia/kernel/kernel_state.cc

```diff
@@ -316,6 +316,18 @@ void KernelState::SetExecutableModule(object_ref<UserModule> module) {
     *variable_ptr = executable_module_->hmodule_ptr();
   }
 
+  // Setup the kernel's ExLoadedImageName field
+  export_entry = processor()->export_resolver()->GetExportByOrdinal(
+      "xboxkrnl.exe", ordinals::ExLoadedImageName);
+
```

Comment:
This empty line is not needed — these two parts set up one thing; while this entire function sets up a lot of others.